model/sparql/GenericSparql.php

```diff
@@ -2229,19 +2228,28 @@ public function listConceptGroupContents($groupClass, $group, $lang,$showDepreca
      * @param string $lang language of labels to return
      * @param int $offset offset of results to retrieve; 0 for beginning of list
      * @param int $limit maximum number of results to return
+     * @param boolean $showDeprecated whether to include deprecated concepts in the change list
      * @return string sparql query
      */
-    private function generateChangeListQuery($prop, $lang, $offset, $limit=200) {
+    private function generateChangeListQuery($prop, $lang, $offset, $limit=200, $showDeprecated) {
         $fcl = $this->generateFromClause();
         $offset = ($offset) ? 'OFFSET ' . $offset : '';
 
+        //Additional clauses when deprecated concepts need to be included in the results
+        $optionVar = ($showDeprecated) ? '?replacedBy' : '';
+        $optionStart = ($showDeprecated) ? 'OPTIONAL {' : '';
+        $optionEnd = ($showDeprecated) ? '}' : '';
+        $deprecatedOption = ($showDeprecated) ? 'OPTIONAL { ?concept dc:modified ?date ; owl:deprecated TRUE ; dc:isReplacedBy ?replacedBy }' : '';
+
         $query = <<<EOQ
-SELECT DISTINCT ?concept ?date ?label $fcl
+SELECT DISTINCT ?concept ?date ?label $optionVar $fcl
```

Comment:
I think you could just use ?replacedBy, which would yield null values if not bound?